pkg/image/importer/importer.go

```diff
@@ -129,6 +132,11 @@ func (i *ImageStreamImporter) importImages(ctx gocontext.Context, retriever Repo
 		repoName := defaultRef.RepositoryName()
 		registryURL := defaultRef.RegistryURL()
 
+		if !importer.isAllowedRegistry(defaultRef.RegistryHostname()) {
```

Comment:
you're right... I thought we want to do this in importer, but this really is a validation... i will pass the list of allowed registries to import validator and verify it there.